### DISCUSSION TITLE
Disable scroll into view on Popover

### DISCUSSION THREAD

**Discussion by ghost** [tailwindlabs/headlessui#1678] [Help]
Hi,

I'm currently using the Popover example provided by TailwindUI ([With dropdown product filters](https://tailwindui.com/components/ecommerce/components/category-filters#component-78dbef177d9356295464384a7cb39506)) and when clicking on one of the button it produces a scroll up to place the panel at the center of the screen.
Is there any way to disable that feature ?

https://user-images.githubusercontent.com/104102859/179239514-bae92aa9-c296-45d0-a127-b175c94824fb.mov

Thank you

**Comment by Syranda** [comment]
Have you found a solution yet? Facing the same problem...

**Reply by ghost** [reply]
Nope but I didn't dig that deep to find an alternative solution, so there might be a way!

**Reply by Syranda** [reply]
Found a little workaround: if the Popover button lives inside a container with `position: fixed` it does not scroll on focus. Only really a workaround if you use it as e.g. navigation
